tests/logger_loguru/test_stack_inspection.py

```diff
@@ -1,56 +0,0 @@
-# Copyright 2010 New Relic, Inc.
```

Comment:
Why did this test get removed?

Reply:
Those can be validated as attributes now instead of the clunky message string check we had before. They're included in `test_loguru_default_context_attributes`. Here's the 2 of them that are checked for value that are equivalent to what was tested before.

```python
            "context.file": "(name='%s', path='%s')" % ("test_attributes.py", str(__file__)),
            "context.function": "test_loguru_default_context_attributes",
```